.github/component_owners.yml

```diff
@@ -20,6 +20,7 @@ components:
     - SergeyKanzhelev
   src/OpenTelemetry.Extensions/:
     - codeblanch
+    - mikegoldsmith
```

Comment:
Please keep in sync with test/OpenTelemetry.Extensions.Tests/

@CodeBlanch, FYI